.github/workflows/r-python-interop-testing.yml

```diff
@@ -2,10 +2,11 @@ name: TileDB-SOMA R-Python interop testing
 
 on:
   pull_request:
-    paths:
-      - "apis/python/**"
-      - "apis/r/**"
-      - "apis/system/**"
+    # Temp force run all
+    #paths:
+    #  - "apis/python/**"
+    #  - "apis/r/**"
+    #  - "apis/system/**"
```

Comment:
Maybe undo these before merge?

Reply:
ok

i'd prefer to keep them for now -- we need _some_ pre-merge signal as the way we found this was a post-merge CI fail. :(

I'll update the comment, with a TODO to maybe revert this later ...

Reply:
I'm confused as to what the intent of this change was, because its effect will be that, rather than running only on reviews which have changes to `apis/{python,r,system}`, it will now run on *every* PR. If it is going to be reverted before merge anyway then it's moot.